power-outage-monitor/README.md

```diff
@@ -0,0 +1,114 @@
+# Power Outage Detector
+
+Use Notecard with a mains-powered USB power brick and LiPo battery to build a globally availalbe power outage detector.
+
+- [Power Outage Detector](#power-outage-detector)
+  - [You will need](#you-will-need)
+  - [Hardware Setup](#hardware-setup)
+  - [Notecard Firmware](#notecard-firmware)
+  - [Cloud Setup](#cloud-setup)
+  - [Notecard Setup](#notecard-setup)
+    - [Configure Notehub Project and Connection Mode](#configure-notehub-project-and-connection-mode)
+    - [Configure Power Outage Alerts](#configure-power-outage-alerts)
+  - [System Test](#system-test)
+    - [Setup](#setup)
+    - [Simulating a Power Outage](#simulating-a-power-outage)
+    - [Blues Wireless Community](#blues-wireless-community)
+
+## You will need
+
+* LiPo battery with JST cable
+* AC to USB Adapter (power brick)
+* USB A to Micro USB cable
+* [Notecarrier-A](https://shop.blues.io/collections/notecarrier/products/carr-al) or [Notecarrier-B](https://shop.blues.io/collections/notecarrier/products/carr-b)
+* Cellular [Notecard](https://blues.io/products/notecard/)
+
+
+## Hardware Setup
+
+1. Assemble Notecard, Notecarrier and LiPo battery as described in our [tutorial](https://dev.blues.io/quickstart/notecard-quickstart/notecard-and-notecarrier-b/).
+
+2. Keep the Notecard connected to the computer with the in-browser terminal active until you have completed [Notecard Setup](#sotecard-setup).
+
+## Notecard Firmware
+
+Your notecard should be running [firmware version 3.5.1](https://dev.blues.io/notecard/notecard-firmware-updates/#v3-5-1-october-7th-2022) or higher version. The simplest way to update firmware is to do an [over-the-air update](https://dev.blues.io/notecard/notecard-firmware-updates/#ota-dfu-with-notehub)
+
+## Cloud Setup
+
+- Sign up for a free account on [Notehub.io](https://notehub.io), and create a new project.
+  `{"req":"hub.set", "product":"com.your-company.your-name:your_product", "mode":"continuous"}`
+- Twillio route: TBD
+
+## Notecard Setup
+
+In these steps, you will use the in-browser terminal to configure your notecard to report power outage events to your Notehub project.
+
+### Configure Notehub Project and Connection Mode
+
+First, we will set the serial number (name) of your notecard and the ProductUID. The ProductUID is the one corresponding to the project you careted in [Cloud Setup](#cloud-setup). Replace the `sn` and `product` values with your own and paste the resulting command into the in-browser terminal.
+
+```json
+{
```

Comment:
Just curious, are you able to paste multi-line requests in the in-browser terminal? I don't think it works for me.

Reply:
yes it does work - but you can only paste on request at a time. The terminal removes the line breaks and sends the pasted text as a single request. I like your approach with the notecard CLI.